### PR TITLE
feat(slack): clean up empty cmd handling

### PR DESCRIPTION
**Pull request by sgandon**
##### ISSUE TYPE
 - Bug fix Pull Request

##### SUMMARY
The empty command `@BotKube` is managed in the executor.go file for having a identic behaviour on all intergrations but the Slack bot is handling it differently.
So I proposed a cleanup in this  PR.

### CODE REVIEW

File: pkg/bot/slack.go [hunk 1]
```diff
@@ -150,10 +150,7 @@ func (sm *slackMessage) HandleMessage(b *SlackBot) {
 	}
 
 	// Trim the @BotKube prefix
-	sm.Request = strings.TrimPrefix(sm.Event.Text, "<@"+sm.BotID+"> ")
-	if len(sm.Request) == 0 {
-		return
-	}
+	sm.Request = strings.TrimPrefix(sm.Event.Text, "<@"+sm.BotID+">")
```
Comment (PrasadG193) [inline pkg/bot/slack.go:153]: Does empty command e.g `@BotKube` return an error message now?

Comment (PrasadG193) [reply pkg/bot/slack.go:153]: @sgandon could you confirm this? Rest looks good

**Comment by sgandon** [comment]
I don't understand but my response is in "pending state" !!!
here is it
Yes it does.
this is managed by the executor : https://github.com/infracloudio/botkube/pull/424/files#diff-fccd3962f0d0da63e38e16963f1ac9ecdc37cbbe33beed7e6aee1d0de163f4e1R183
This way there is a common behaviour between all integrations

**Comment by PrasadG193** [comment]
Cool! Thanks @sgandon

**Review by PrasadG193 — APPROVED** [review]

**Pull request merged**